package.js

```diff
@@ -0,0 +1,50 @@
+Package.describe({
+  name: 'space:performance',
+  version: '0.1.0',
+  summary: 'Suite of tools to test and measure the performance of your Space system',
+  git: 'https://github.com/meteor-space/performance.git',
+  documentation: 'README.md'
+});
+
+Package.onUse(function(api) {
+
+  api.versionsFrom('1.2.1');
```

Comment:
Do we need the latest Meteor version? We sticked to 1.0.0 for most other packages, so that app devs are not forced to the latest if not necessary :wink: